mchf-eclipse/drivers/audio/cw/cw_gen.c

```diff
@@ -226,7 +226,7 @@ static const float sm_table[CW_SMOOTH_TBL_SIZE] =
 
 void CwGen_SetSpeed()
 {
-    ps.dit_time         = 1650/ts.keyer_speed;      //100;
+    ps.dit_time         = 1920/ts.keyer_speed;      //100;
 }
```

Comment:
I have checked with my oscilloscope and measured 100ms at 12 WPM and 50ms at 24 WPM. Maybe my MCU is running faster? Tomorrow I will check the slow speeds.
73, Erwin